src/utils/num.ts

```diff
@@ -97,28 +169,58 @@ export function assertInRange(
 
 /**
  * Convert BigNumberish array to decimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: decimal string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToDecimalStringArray(bigNumberishArray);
+ * // result = ["123", "456", "789"]
+ * ```
  */
 export function bigNumberishArrayToDecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toBigInt(x).toString(10));
 }
 
 /**
  * Convert BigNumberish array to hexadecimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: hex-string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToHexadecimalStringArray(bigNumberishArray);
+ * // result = ["0x7b", "0x1c8", "0x315"]
+ * ```
  */
 export function bigNumberishArrayToHexadecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toHex(x));
 }
 
 /**
  * Test if string is whole number (0, 1, 2, 3...)
+ * @param value The string to be tested.
+ * @return {boolean} Returns true if the value is a number, otherwise returns false.
```

Comment:
@return is not existing in JSDoc.
Only @returns (with a `s`) is authorized : https://jsdoc.app/tags-returns

Reply:
To check everywhere

Reply:
fixed, changed all @return to @returns and conflict resolved